kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -117,35 +203,118 @@
   @require '~kolibri.styles.coreTheme'
   @require '~kolibri.styles.navBarItem'
 
+  $headerheight = 57px
+  $footerheight = 100px
+
   .nav-wrapper
-    display: table
-    table-layout: fixed
+    top: 0px
     background: $core-bg-light
     font-weight: 300
     position: fixed
-    z-index: auto
-    @media screen and (min-width: $portrait-breakpoint + 1)
-      font-size: 1em
-      height: 100%
-      width: $nav-width
-    @media screen and (max-width: $portrait-breakpoint)
-      font-size: 0.8em
-      bottom: 0
-      width: 100%
-      min-width: 300px
+    z-index: 1001
+    font-size: 1em
+    height: 100vh
+    width: $nav-width
+    overflow: auto
+    -webkit-overflow-scrolling: touch
+    .logo
+      margin: auto
+      display: block
+      height: 125px
+      width: @height
 
   .nav-main
     background: $core-bg-light
-    height: 100vh
-    @media screen and (max-width: $portrait-breakpoint)
-      display: table-row
-      height: $nav-portrait-height
+    max-width: $nav-width
 
   a.active:focus svg
     fill: $core-bg-light
 
-  .nav-icon
-    width: 40px
-    height: 40px
+  .header
+    overflow: auto
+    overflow-y: hidden
+    background-color: $core-text-default
+    height: $headerheight
+    .logo, .title, .close
+      float: left
+    .title, .close
+      color: $core-bg-light
+    .close
+      font-size: ($headerheight / 2)
+      top: 50%
+      transform: translateY(-50%)
+      position: relative
+      margin-right: ($nav-width/20)
+      margin-left: ($nav-width/20)
+      border: none
+    .title
+      font-size: ($headerheight / 3)
+      font-weight: bold
+    .logo
+      width: $headerheight
+      height: @width
+      margin-right: ($nav-width/20)
+
+  .footer
+    bottom: 0
+    position: absolute
+    overflow: auto
+    overflow-y: hidden
+    background-color: $core-text-default
+    height: $footerheight
+    width: $nav-width
+    .logo, .message-container
+      top: 50%
+      transform: translateY(-50%)
+      position: relative
+    .logo
+      float: left
+      width: ($footerheight/2)
+      height: @width
+      margin-right: ($nav-width/20)
+      margin-left: ($nav-width/20)
+    .message-container
+      .message
+        color: $core-bg-light
+        font-size: ($footerheight/10)
+
+  .modal-overlay
+    position: fixed
+    top: 0
+    left: 0
+    width: 100%
+    height: 100%
+    background: rgba(0, 0, 0, 0.7)
+    transition: opacity 0.3s ease
+    background-attachment: fixed
+    z-index: 10
+
+</style>
+
+<style lang="stylus">
 
+  @require '~kolibri.styles.coreTheme'
+
+  // Customize Keen UI Menu option
+  .nav-main
+    .ui-menu-option
+      margin: 5px 0px
+      &:not(.is-divider)
+        font-size: 1.2em
```

Comment:
I'm not sure if we should be using `em`s for fonts - it's resulting in weird, non-standard font sizes:

![image](https://cloud.githubusercontent.com/assets/2367265/22995963/d198286a-f381-11e6-8357-8d774639094d.png)

![image](https://cloud.githubusercontent.com/assets/2367265/22996046/2b1b5394-f382-11e6-8098-5dcacc26a623.png) ![image](https://cloud.githubusercontent.com/assets/2367265/22996038/22c8f674-f382-11e6-9fb2-eb1be5c7eed5.png)

https://material.io/guidelines/patterns/navigation-drawer.html#navigation-drawer-specs

However I understand that px is not considered accessible, so I'm not sure what the right answer is here.